App Assurance Program (#104)
AdobeDocs/commerce-marketplace: src/pages/guides/sellers/assurance.md

---
title: App Assurance Program

> description: This page describes the App Assurance Program for marketplace apps and extensions for Adobe Commerce.

Reviewer: ```suggestion
description: Learn about the App Assurance Program for Marketplace apps and extensions for Adobe Commerce.
```

## Review process

After passing our [Extension Quality Program](./extension-quality-program.md), submissions to the App Assurance Program undergo a thorough code review, Magento Functional Testing Framework (MFTF) testing, and performance testing. The following sections provide a description of each step of the verification process.

### Code review

The enhanced code review consists of a static code review and a security code review. The static code review consists of the following:

- Review the code for any bugs
- Validate that the submission adheres to the [Magento coding standard](https://github.com/magento/magento-coding-standard)
- Validate the change level against the level submitted by the developer
- Check dependencies for any known vulnerabilities
- Confirm that the listing is compatible with the newest version of Adobe Commerce

The security code review consists of the following:

- Check for outdated frameworks with known vulnerabilities
- Check for possible SQL injection and data sanitization issues
- Check databases for proper structure

> - Ensure proper uses of classes, methods, etc

Reviewer: ```suggestion
- Ensure proper usage of classes and methods
```

etc requires a period, and that wouldn't be parallel, so....
